.github/workflows/documentation.yaml

```diff
@@ -1,43 +1,27 @@
 name: "Documentation"
-# This workflow is outdated and needs to be replaced with a one that
-# uses the new documntation location.
 
 on:
-  schedule:
-    - cron: "0 0 * * *"
+  push:
+    paths:
+      - docs/**
+      - mkdocs.yml
+      - README.md
+    branches:
+      - main
   workflow_dispatch:
```

Comment:
Looks like `workflow_dispatch` can be removed completely.

Reply:
We spoke about this offline with Tomas. The workflow dispatch is still valid if we want a workflow to be triggered manually.